contracts/ECDSASign.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract ECDSASign is Ownable {
+    address private _signer;
+
+    constructor(address signer_) {
+        require(signer_ != address(0), "ECDSASign: signer_ is zero address");
+        _signer = signer_;
+    }
+
+    /// @notice verifySignature validates a message by ecrecover to ensure
+    //          it is signed by signer.
+    /// @param message_ - the raw message for signing
+    /// @param r_ - part of signature for validating parameters integrity
+    /// @param s_ - part of signature for validating parameters integrity
+    /// @param v_ - part of signature for validating parameters integrity
+    function verifySignature(
```

Comment:
```suggestion
    function isValidSignature(
```

Reply:
I take this name form Hieu. we change this naming back and force. @jollyjoker992 do you have strong opinion here?

Reply:
I think `isXX` return bool. `verfifyXX` can just check and revert like `validateSaleData` above

Reply:
@lpopo0856 No, i dont. `validSignature()` or `isValidSignature` could be ok. 
BTW, i used to name a function return bool without `is` but it's fine, it's up to you.